modules/claims_api/config/schemas/v2/2122a.json

```diff
@@ -0,0 +1,373 @@
+              "description": "Street address with number and name.",
+              "type": "string",
+              "pattern": "^([-a-zA-Z0-9'.,&#]([-a-zA-Z0-9'.,&# ])?)+$",
+              "maxLength": 30
+            },
+            "aptUnitNumber": {
+              "type": "integer",
+              "maxLength": 5
+            },
+            "city": {
+              "description": "City for the address.",
+              "type": "string",
+              "example": "Portland",
+              "maxLength": 18
+            },
+            "state": {
+              "description": "State for the address.",
+              "type": "string",
+              "example": "OR",
+              "maxLength": 2
+            },
+            "country": {
+              "description": "Country of the address.",
+              "type": "string",
+              "example": "US",
+              "maxLength": 2
+            },
+            "zipFirstFive": {
+              "description": "Zipcode (First 5 digits) of the address.",
+              "type": "string",
+              "pattern": "^\\d{5}?$",
+              "example": "12345"
+            },
+            "zipLastFour": {
+              "description": "Zipcode (Last 4 digits) of the address.",
+              "type": "string",
+              "pattern": "^\\d{4}?$",
+              "example": "6789"
+            }
+          }
+        },
+        "phone": {
+          "$comment": "the phone fields must not exceed 20 chars, when concatenated",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "areaCode",
+            "phoneNumber"
+          ],
+          "properties": {
+            "countryCode":     { "type": "string", "pattern": "^[0-9]+$" },
+            "areaCode": {
+              "description": "Area code of the phone number.",
+              "type": "string", "pattern": "^[2-9][0-9]{2}$",
+              "example": "555"
+            },
+            "phoneNumber": {
+              "description": "Phone number.",
+              "type": "string", "pattern": "^[0-9]{1,14}$",
+              "example": "555-5555"
+            },
+            "phoneNumberExt":  { "type": "string", "pattern": "^[a-zA-Z0-9]{1,10}$" }
+          }
+        },
+        "email": {
+          "description": "Email address of the claimant.",
+          "type": "string",
+          "pattern": ".@.",
+          "maxLength": 61,
+          "example": "claimant@example.com"
+        },
+        "relationship": {
+          "description": "Relationship of claimant to the veteran.",
+          "type": "string",
+          "example": "Spouse"
+        }
+      }
+    },
+    "representative": {
+      "description": "Details of the individual representative representing the veteran.",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "poaCode",
+        "firstName",
+        "lastName",
+        "type",
+        "address"
```

Comment:
we can probably change this list once we get registration number. We'll be able to get info from the rep record and org records associated with the ids coming in.